single_tech_samples/fabric/feature_engineering_on_fabric/README.md

```diff
@@ -0,0 +1,369 @@
+  
+  ![data_pipeline_10](./images/data_pipeline/data_pipeline_10.png)
+
+### Notebook activities
+
+Once the 'Data Landing' activity is configured, we can start to add more activities to perform the data ingestion, cleansing, transformation, and feature set registration tasks. Each of these tasks is performed by a 'notebook' activity which invokes the corresponding notebook:
+
+- [data_ingestion](./src/notebooks/data_ingestion.ipynb)
+- [data_cleansing](./src/notebooks/data_cleansing.ipynb)
+- [data_transformation](./src/notebooks/data_transformation.ipynb)
+- [feature_set_registration](./src/notebooks/feature_set_registration.ipynb)
+
+When we finish configuring the whole pipeline, it should look like this.
+
+![data_pipeline_11](./images/data_pipeline/data_pipeline_11.png)
+
+Here are the steps involved in configuring these notebook activities.
+
+- Add four 'Notebook' activities
+
+  Select 'Activities' tab and add four 'Notebook' activities.
+
+  ![data_pipeline_12](./images/data_pipeline/data_pipeline_12.png)
+
+- Link the activities
+
+  Link the activities within the pipeline. Make sure to drag the 'On success' check mark located on the right side of each activity to the next one. It would ensure that the subsequent activity will be executed only when the current one has completed successfully.
+
+  ![data_pipeline_13](./images/data_pipeline/data_pipeline_13.png)
+
+- Configure the 'Notebook' activities
+
+  Rename each notebook activity by selecting it, clicking 'General' tab and updating the name of the 'Notebook' activity. Name them as 'Data Ingestion', 'Data Cleansing', 'Data Transformation', and 'Feature Registration' respectively.
+
+  For each notebook activity, add the following 'Base parameters' in the Settings' tab:
+  ```text
+  Name: client_secret
+  Type: String
+  Value: `@pipeline().parameters.client_secret`
+  ```
+
+- Associate actual notebooks with the activities
+
+  Within the 'Settings' tab, select our current workspace and then choose the right Notebook file.
+
+  ```text
+  Data Ingestion       -> data_ingestion
+  Data Cleansing       -> data_cleansing
+  Data Transformation  -> data_transformation
+  Feature Registration -> feature_set_registration
+  ```
+  
+  ![data_pipeline_15](./images/data_pipeline/data_pipeline_15.png)
+
+## Execute and verify
+
+Once the setup is complete, you can trigger the pipeline and verify the results.
+
+### Trigger the data pipeline
+
+Access the Fabric workspace and locate the item categorized as a `Data pipeline`.
+
+![data pipeline](./images/pipeline_image.png)
+
+In order to trigger the data pipeline, choose this data pipeline, and click `Run`.
+
+![data pipeline](./images/data_pipeline.png)
+
+In the pane, you can check the input and output, which gives the more detailed execution result.
+
+![pipeline run](./images/pipeline_run.png)
+
+### Verify the data lineage in Purview
+
+TBD
+
+### Verify the features in Feature Store
+
+TBD
```

Comment:
```suggestion
If the pipeline executes successfully, you can verify the features registered in the Azure ML Managed Feature Store.
<img width="949" alt="image" src="https://github.com/Azure-Samples/modern-data-warehouse-dataops/assets/44962887/97cf6291-65e6-400a-92d8-50263777933f">
```

Reply:
Thanks Siliag, can we also add some additional description of what it means.